routers/web/web.go

```diff
@@ -757,6 +757,7 @@ func registerRoutes(m *web.Route) {
 		m.Group("/actions", func() {
 			m.Get("", admin.RedirectToDefaultSetting)
 			addSettingsRunnersRoutes()
+			addSettingsSecretsRoutes()
```

Comment:
I don't think we can reuse this routers function and the previous usage are also wrong. At least, the `RedirectLink` will always be the repository, so it is not right.

Reply:
For that PR, I repected how it was done for the other types of secrets. All the logic about secrets resides in the repo/settings/secrets.go file. As the RedirectLink is being set depending on the context, it works fine (just tried)